src/main/java/com/conveyal/gtfs/graphql/fetchers/PolylineFetcher.java

```diff
@@ -0,0 +1,87 @@
+package com.conveyal.gtfs.graphql.fetchers;
+
+import com.conveyal.gtfs.graphql.GTFSGraphQL;
+import com.conveyal.gtfs.util.PolylineUtils;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import org.apache.commons.dbutils.DbUtils;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Point;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * GraphQL fetcher to get encoded polylines for all shapes in a GTFS feed.
+ */
+public class PolylineFetcher implements DataFetcher {
+    public static final Logger LOG = LoggerFactory.getLogger(PolylineFetcher.class);
+
+    @Override
+    public Object get(DataFetchingEnvironment environment) {
+        GeometryFactory gf = new GeometryFactory();
+        List<Shape> shapes = new ArrayList<>();
+        Map<String, Object> parentFeedMap = environment.getSource();
+        String namespace = (String) parentFeedMap.get("namespace");
+        Connection connection = null;
+        try {
+            connection = GTFSGraphQL.getConnection();
+            Statement statement = connection.createStatement();
+            String sql = String.format(
+                "select shape_id, shape_pt_lon, shape_pt_lat from %s.shapes order by shape_id",
+                namespace
+            );
+            LOG.info("SQL: {}", sql);
+            if (statement.execute(sql)) {
```

Comment:
Seems like this will always fetch every single shape and not do any filtering according to shape_id or bounding box. Either that logic should be implemented here, or the args in the GraphQL schema should be removed.